src/syft/lib/torchvision/__init__.py

```diff
@@ -13,6 +14,7 @@
 from .allowlist import allowlist
 
 TORCHVISION_VERSION = version.parse(tv.__version__)
+TORCH_VERSION = version.parse(th.__version__)
```

Comment:
Shouldnt we match the torchvision capabilities to the torchvision version? Occasionally there is more than 1 torchvision version for a torch version like 0.8.0 and 0.8.1 due to a bug fixes.

Reply:
I tried comparing the Torchvision version but I was facing some issues, I'll re-check